cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -74,6 +74,7 @@ data:
         server_name _;
         root /var/www;
         index index.html;
+        large_client_header_buffers 4 32k;
```

Comment:
Should the size be configurable via values?

Reply:
Agreed, we should follow up with improved flexibility.  Hopefully this first step alleviates the need for you to maintain a fork.